src/sinol_make/commands/run/__init__.py

```diff
@@ -191,8 +193,14 @@ def compile(self, solution):
             self.COMPILATION_DIR, "%s.compile_log" % self.extract_file_name(solution))
         source_file = os.path.join(os.getcwd(), "prog", self.get_solution_from_exe(solution))
         output = os.path.join(self.EXECUTABLES_DIR, self.get_executable(solution))
+
+        weak_compilation_flags = False
+        if hasattr(self, "args") and self.args.weak_compilation_flags:
+            weak_compilation_flags = True
```

Comment:
```suggestion
        weak_compilation_flags = hasattr(self, "args") and self.args.weak_compilation_flags
```

Reply:
Also, can `self` not have `args`?

Reply:
Changed 7d050b88495eb66ddda18314deefaeca167f4b01